packages/reactor-browser/src/hooks/use-on-drop-file.ts

```diff
@@ -1,23 +1,23 @@
-import type { FileNode } from "document-drive";
-import { useCallback } from "react";
-import { addFileWithProgress } from "../actions/document.js";
 import type {
   ConflictResolution,
   FileUploadProgressCallback,
-} from "../types/upload.js";
+} from "@powerhousedao/reactor-browser";
+import type { FileNode } from "document-drive";
+import { useCallback } from "react";
+import { addFileWithProgress } from "../actions/document.js";
 import { useSelectedDriveSafe } from "./drives.js";
 import { useSelectedFolder } from "./nodes.js";
+import { useSupportedDocumentTypes } from "./reactor.js";
 
-type UseOnDropFile = (
-  documentTypes?: string[],
-) => (
+type UseOnDropFile = () => (
   file: File,
   onProgress?: FileUploadProgressCallback,
   resolveConflict?: ConflictResolution,
 ) => Promise<FileNode | undefined>;
 
-export const useOnDropFile: UseOnDropFile = (documentTypes = []) => {
+export const useOnDropFile: UseOnDropFile = () => {
   const [selectedDrive] = useSelectedDriveSafe();
+  const supportedDocumentTypes = useSupportedDocumentTypes();
```

Comment:
This is not the same logic. The supported document types are defined for each drive, while this hooks returns all the document models in the reactor.
For example, Vetra only allows dropping vetra documents.
This hook should accept a `documentTypes` prop that defaults to `useSupportedDocumentTypes`.